vllm/lora/ops/sgmv_shrink.py

```diff
@@ -5,48 +5,61 @@
 https://arxiv.org/abs/2310.18547
 """
 
+from typing import List
+
 import torch
 import triton
 import triton.language as tl
 
 from vllm.utils import direct_register_custom_op
 
+from .utils import _get_lora_a_ptr
+
 
 @triton.jit
 def _sgmv_shrink_kernel(
-    input_ptr,
-    lora_ptr,
-    out_ptr,
-    N,
-    K,
-    b_seq_start_loc,
-    seq_lens,
-    lora_indices,
-    scaling,
-    xm_stride,  # hidden_size
-    xk_stride,  # 1
-    l0_stride,  # hidden_size*max_rank
-    lora_k_stride,
-    lora_n_stride,
-    cm_stride,
-    cn_stride,
-    BLOCK_M: tl.constexpr,
-    BLOCK_N: tl.constexpr,
-    BLOCK_K: tl.constexpr,
-    EVEN_K: tl.constexpr,
-    SPLIT_K: tl.constexpr,
-):
+        input_ptr,
+        lora_ptr,  #1-3
+        out_ptr,
+        N,
+        K,
+        b_seq_start_loc,
+        seq_lens,
+        lora_indices,
+        scaling,
+        xm_stride,  # hidden_size
+        xk_stride,  # 1
+        ls_d0_ptr,  # hidden_size*max_rank
+        ls_d1_ptr,
+        ls_d2_ptr,
+        c0_stride,
+        cm_stride,
+        cn_stride,
```

Comment:
nit: maybe in a followup PR, it'd be good to standardize on the naming the stride parameters. i.e. `cm_stride` -> `c1_stride`, `cn_stride` -> `c2_stride`, `xm_stride` -> `x1_stride`, `xk_stride` -> `x2_stride` . same with the expand kernel.

Reply:
Good idea, I'll make the modifications later.

Reply:
Done